client/src/includes/userbar.ts

```diff
@@ -11,6 +11,55 @@ This component implements a roving tab index for keyboard navigation
 Learn more about roving tabIndex: https://w3c.github.io/aria-practices/#kbd_roving_tabindex
 */
 
+export const sortAxeViolations = (violations: AxeResults['violations']) => {
+  interface inaccessibleNode {
+    selectedNode: HTMLElement;
+    selectorName: string;
+  }
+  const inaccessibleNodes: inaccessibleNode[] = [];
+
+  violations.forEach((violation) => {
+    violation.nodes.forEach((node) => {
+      const selectorName = node.target[0];
+      const selectedNode = document.querySelector<HTMLElement>(selectorName);
+      if (!selectedNode) return;
+      // Push both node (to compare DOM position) and selector (to sort Axe results)
+      inaccessibleNodes.push({
+        selectedNode,
+        selectorName,
+      });
+    });
+  });
+
+  const sortedNodes = inaccessibleNodes.sort((a, b) =>
```

Comment:
Our linting is going to complain about such short identifiers.

```suggestion
  const sortedNodes = inaccessibleNodes.sort((nodeA, nodeB) =>
```